libs/feat/design-system/components/templates/layouts/SideNavBarLayout/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from "react";
+import { View } from "@/ui-kit/ui/view";
+import { Button } from "../../../atoms/Fields/Button";
+import { Drawer, DrawerBackdrop, DrawerContent } from "@/ui-kit/ui/drawer";
+import {
+  NavigationSideMenu,
+  type NavigationSideMenuProps,
+} from "../../../organisms/NavigationSideMenu";
+
+export const SideNavBarLayout: React.FC<{
+  children: React.ReactNode;
+  navigationMenu: NavigationSideMenuProps;
+}> = ({ children, navigationMenu }) => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 800);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 800);
+      if (window.innerWidth > 800) {
+        setIsMenuOpen(false);
+      }
+    };
+
+    window.addEventListener("resize", handleResize);
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  if (isMobile)
```

Comment:
Warning: please be very explicit and careful with isMobile
I mean, the app can run on web and expo; so this variable didn't describe if it's mobile but if it's mobile sizing/viewport

```suggestion
  if (isMobileViewport)
```